src/main/java/com/southernartsoap/model/Cart.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package com.southernartsoap.model;
+
+/**
+ *
+ * @author jared
+ */
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+public class Cart {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "cart_id")
+    private Long id;
+    
+    @OneToOne(fetch = FetchType.LAZY, cascade = CascadeType.ALL) //does this map to user or usermodel?
+    private Long customerId;
```

Comment:
since we have a user model instead of customer, should this be userId? also, still not familiar with fetch, but does it need some sort of mapping to User??

Reply:
fetch has to do with when the database loads it. Lazy = on demand. Eager = ???. All the examples I see use Lazy so I figure that's what we should do until something doesn't work. 

The schema labeled is customerId. I'm not clear if customerId is purely an internal name or if it's also what goes into the database column. For example, could we name it private Long aokdjlkajd; and have it the user id? 

The OneToOne is the mapping in the User.java.